packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -305,6 +305,36 @@ describe('I18n', () => {
       expect(i18n.formatNumber(0.12345, options)).toBe(expected);
     });
 
+    it('updates format on multiple calls', () => {
```

Comment:
Do we need to test that it does not create new object if same arguments are called twice?

Reply:
I don't this is a particularly testable case. The instance of `Intl` is outside the scope of this class and would be considered private.